src/java.base/share/classes/sun/net/www/MessageHeader.java

```diff
@@ -50,6 +50,7 @@ public MessageHeader () {
         grow();
     }
 
+    @SuppressWarnings("this-escape")
```

Comment:
An alternative here could be to make the class final. AFAICS it's not subclassed anywhere. If you'd prefer not to do this here then maybe a followup issue could be logged?

Reply:
I'd prefer if that kind of change were done as a subtask of

[JDK-8325263](https://bugs.openjdk.org/browse/JDK-8325263): Address this-escape lint warnings java.base (umbrella)

Reply:
Thanks Joe. I logged [JDK-8325361](https://bugs.openjdk.org/browse/JDK-8325361): Make sun.net.www.MessageHeader final